### ISSUE TITLE
WebUI not displaying metrics properly

### ISSUE THREAD

**Issue by xquyvu** [allegroai/clearml#1111]
## Describe the bug

WebUI is not showing all scalars when there are many metrics. As you can see in the screenshot, my horizontal scrollbar is scrolled all the way to the left. The metrics can be seen if I export them to csv.

![image](https://github.com/allegroai/clearml/assets/40758906/344bf621-c857-4960-81ba-200800e0c017)

## To reproduce

```
logger = task.get_logger()

for _ in range(20):
    logger.report_single_value('very very long metric', 0.00000000000000009)
```

## Expected behaviour
I should be able to scroll all the way to the left to see all metrics.

## Environment
* Server type: app.clear.ml
* ClearML SDK Version: 1.2
* Python Version: 3.11
* OS: Macos
* 
## Related Discussion
If this continues a slack thread, please provide a link to the original slack thread: N/A

**Comment by xquyvu** [comment]
Slightly related to this, the parallel coordinates chart also struggles with long metric names

![image](https://github.com/allegroai/clearml/assets/40758906/19f6a041-9426-4b6b-ae32-4db9b30f453d)

**Comment by ainoam** [comment]
Thanks for reporting @xquyvu 

Hope to have a fix out soon.

**Comment by pollfly** [comment]
Hey @xquyvu! Just letting you know that the parallel coordinate plot issue has been resolved in the recently released [clearml-server 1.13.0](https://github.com/allegroai/clearml-server/releases/tag/v1.13.0). Let us know if there are any issues :)

**Comment by xquyvu** [comment]
awesome, thanks, it's working for me :)
